wiki/Tournaments/CIC/2022/en.md

```diff
@@ -294,14 +294,10 @@ Sunday, 20 March 2022:
 | Bracket | Player 1 |  |  | Player 2 | Match link |
 | :-: | --: | :-: | :-: | :-- | :-- |
 | Lower | [Mamat](https://osu.ppy.sh/users/3031177) ![][flag_ID] | 1 | **6** | ![][flag_ID] **[Keisuu](https://osu.ppy.sh/users/3856673)** | [#1](https://osu.ppy.sh/community/matches/98923798) |
-| Lower | **[Ruu](https://osu.ppy.sh/users/3212755)** ¹ ![][flag_ID] | **1** | 0 | ![][flag_ID] ² [Minalinsky-](https://osu.ppy.sh/users/2823883) | [#1](https://osu.ppy.sh/community/matches/98926575) |
+| Lower | **[Ruu](https://osu.ppy.sh/users/3212755)** [^default-winner] ![][flag_ID] | **1** | 0 | ![][flag_ID] [^default-loser] [Minalinsky-](https://osu.ppy.sh/users/2823883) | [#1](https://osu.ppy.sh/community/matches/98926575) |
```

Comment:
it's usually done like this instead of having a match link (https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE.md?plain=1#L131)

```suggestion
| Lower | **[Ruu](https://osu.ppy.sh/users/3212755)** ![][flag_ID] | **1** | 0 | ![][flag_ID] [Minalinsky-](https://osu.ppy.sh/users/2823883) | *win by default* |
```